wordpress-mobile/WordPress-iOS
# Stats: Subscribers List Card

Partially implements #23046, details view will be implemented in another PR.
Related WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/797

Adding Subscribers List Card to Subscribers tab, by using the same design as `Insights -> Total Followers` card and consolidating WordPress.com and Email subscribers.

## Solution

1. Added `subscribersList` to `SubscribersStore` by combining email and dotcom subscribers just as it's done on the web p1713873041361109-slack-C06BR07TJHK
2. Built `subscribersListRows` using the same row as in `Insights`
3. Renamed strings that are necessary for subscribers list
4. Updated tests

## To test:

1. Enable `Stats Traffic and Subscribers tabs` feature flag
2. Open Stats
3. Open Subscribers
5. Confirm that `Subscribers` card appears
6. Confirm that "View more" button exists for sites with more than 6 subscribers
7. Compare with designs IqhXWz3Iir7RMb5XH5gGfZ-fi-883%3A4006

<img width="332" alt="image" src="https://github.com/wordpress-mobile/WordPress-iOS/assets/4062343/83996147-1fbf-48c9-aee2-9067cf65da37">


## Regression Notes
1. Potential unintended areas of impact

None

2. What I did to test those areas of impact (or what existing automated tests I relied on)

3. What automated tests I added (or what prevented me from doing so)

Unit tests

PR submission checklist:


Testing checklist:
